metaflow/current.py

```diff
@@ -6,28 +6,42 @@ def __init__(self):
         self._run_id = None
         self._step_name = None
         self._task_id = None
+        self._retry_count = None
         self._origin_run_id = None
         self._namespace = None
         self._username = None
         self._is_running = False
 
     def _set_env(self,
-                 flow_name,
-                 run_id,
-                 step_name,
-                 task_id,
-                 origin_run_id,
-                 namespace,
-                 username):
+                 flow_name=None,
+                 run_id=None,
+                 step_name=None,
+                 task_id=None,
+                 retry_count=None,
+                 origin_run_id=None,
+                 namespace=None,
+                 username=None,
+                 is_running=True):
```

Comment:
I wonder if its worth having the default to False? Any rationale why you chose True instead?
For now - it seems in both places you're being explicit. Another option might be to leave out any default or set to None. Thoughts?

Reply:
It's set to `True` for parity with nflx-metaflow

Reply:
OK.